roles/st2/tasks/main.yml

```diff
@@ -60,6 +60,9 @@
     loop_var: _conf_section
   tags: st2, config
 
+- name: Configure the StackStorm datastore
+  include: datastore.yaml
+
```

Comment:
Just a final cosmetic request: please include `tags` for this section as used in other blocks.

Reply:
done